generators/lua/loops.js

```diff
@@ -34,75 +34,75 @@ Blockly.Lua.CONTINUE_STATEMENT = 'goto continue\n';
  * @return {string} Generated label or '' if unnecessary
  * @private
  */
-Blockly.Lua.addContinueLabel_ = function(branch) {
-  if (branch.indexOf(Blockly.Lua.CONTINUE_STATEMENT) !== -1) {
+Lua.addContinueLabel_ = function(branch) {
```

Comment:
This should probably be converted to a local variable similar to getListIndex, I think?

Reply:
Good catch. Fixed.